client/ayon_houdini/plugins/publish/collect_render_products.py

```diff
@@ -166,6 +166,15 @@ def get_aov_identifier(self, render_product):
             str: The AOV identifier
 
         """
+        # Allow explicit naming through custom attribute on the render product
+        aov_identifier = render_product.GetPrim().GetAttribute(
+            "ayon:aov_identifier").Get()
```

Comment:
it was always returning `None` on my side because my wrangle was time dependent 
```suggestion
            "ayon:aov_identifier").Get(time=0)
```

![image](https://github.com/user-attachments/assets/ac1a9fc2-f889-4323-8fde-7e08426fe5be)

Reply:
Alternatively, we can use the earliest time.
but keep in mind, we still need to add this 👇 import at the top.
```python
from pxr import Usd
```

```suggestion
            "ayon:aov_identifier").Get(time=Usd.TimeCode.EarliestTime())
```